src/client/v2/algod/algod.ts

```diff
@@ -24,11 +24,37 @@ import {
   CustomTokenHeader,
 } from '../../urlTokenBaseHTTPClient';
 
+/**
+ * Algod client connects an application to the Algorand blockchain. The algod client requires a valid algod REST endpoint IP address and algod token from an Algorand node that is connected to the network you plan to interact with.
+ *
+ * Algod is the main Algorand process for handling the blockchain. Messages between nodes are processed, the protocol steps are executed, and the blocks are written to disk. The algod process also exposes a REST API server that developers can use to communicate with the node and the network. Algod uses the data directory for storage and configuration information.
+ *
+ * #### Relevant Information
+ * [How do I obtain an algod address and token?](https://developer.algorand.org/docs/archive/build-apps/setup/?from_query=algod#how-do-i-obtain-an-algod-address-and-token)
+ *
+ * [Run Algod in Postman OAS3](https://developer.algorand.org/docs/rest-apis/restendpoints/?from_query=algod#algod-indexer-and-kmd-rest-endpoints)
+ */
 export default class AlgodClient extends ServiceClient {
   /**
    * Create an AlgodClient from
    * * either a token, baseServer, port, and optional headers
    * * or a base client server for interoperability with external dApp wallets
+   *
+   * #### Example
+   * ```typescript
+   * const token  = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+   * const server = "http://localhost";
+   * const port   = 4001;
+   * const algodClient = new algosdk.Algodv2(token, server, port);
+   * ```
+   * @remarks
+   * The above configuration is for a sandbox private network.
+   * For applications on production, you are encouraged to run your own node, or use an Algorand REST API provider with a dedicated API key.
```

Comment:
Given the recent outage, I've changed the base server example from algoexplorer to sandbox and added the remarks. Can you see if this is okay? @jasonpaulos